autonomy/cli/scaffold_fsm.py

```diff
@@ -1064,12 +1064,13 @@ class BehaviourTestsFileGenerator(AbstractFileGenerator):
             {rounds},
         )
 
-        from tests.conftest import ROOT_DIR
-        from tests.test_skills.test_abstract_round_abci.test_tools.base import (
+        from packages.valory.skills.abstract_round_abci.test_tools.base import (
             FSMBehaviourBaseCase,
         )
 
 
+        PATH_TO_SKILL = Path(*{FSMName}BaseBehaviour.__module__.split(".")[:-1]).absolute()
```

Comment:
could probably make it `Path({author} / "skills" / {skill_name}).absolute()` which would be easier to comprehend